tests/completion/snippets/task.yaml

```diff
@@ -20,4 +20,5 @@ execute: |
   #shellcheck disable=SC1090
   source "${SPREAD_PATH}/${SPREAD_SUITE}/${SPREAD_VARIANT}.vars"
   export _OUT0 _OUT1 _OUT2 _KEY1 _KEY2 _COMP
+  export XDG_DATA_DIRS="${XDG_DATA_DIRS:-}${XDG_DATA_DIRS:+:}/var/lib/snapd/desktop"
   sudo PATH="$PATH" -E -u test expect -d -f "$d"/task.exp
```

Comment:
The tests use `expect` which is not really possible to use on Ubuntu Core. I will look into writing some tests without using `expect` on Wednesday.

Reply:
There are people who use snapd on Yocto, where read only rootfs is quite commonly used. OTOH we haven't had any reports where completions would break such installs. I guess it has to do with mostly command line tools shipping completions (if at all), while those systems, much like Ubuntu Core are not really intended to be accessed directly though ssh or similar. So I'd say it's ok to address this in a followup, though with the new completions location it shouldn't be a problem at all (unless they ship old bash-completions but then we can always push back).

Reply:
There is now `tests/core/bash-completion` which tests completion on UC22.